bin/Makefile.am

```diff
@@ -60,25 +60,14 @@ regenerate: \
 
 ## Create compatibility symlinks if available
 
-xpmroot_name =`echo "xpmroot" | "$(SED)" -e "${transform}"`
 fvwm_root_name =`echo "fvwm-root" | "$(SED)" -e "${transform}"`
 FvwmCommand_name =`echo "FvwmCommand" | "$(SED)" -e "${transform}"`
 
 install-exec-local:
-	@rm -f $(DESTDIR)$(bindir)/$(xpmroot_name)$(EXEEXT)
-	@$(LN_S) $(fvwm_root_name)$(EXEEXT) \
-		$(DESTDIR)$(bindir)/$(xpmroot_name)$(EXEEXT) || \
-		echo "Minor warning: $(bindir)/$(xpmroot_name)$(EXEEXT) symlink was not created"
 
 install-data-local:
```

Comment:
Ditto here -- if `install-datal-local` is just for `xpmroot` then this line can go as well.

Reply:
I would assume we can move the stanzas. I was unsure if we should, or to keep them there. I will remove them and test.